.cbmc-batch/jobs/aws_cryptosdk_enc_ctx_serialize/aws_cryptosdk_enc_ctx_serialize_harness.c

```diff
@@ -27,24 +27,28 @@ void array_list_item_generator(struct aws_array_list *elems) {
     for (size_t index = 0; index < elems->length; ++index) {
         struct aws_hash_element *val = (struct aws_hash_element *)((uint8_t *)elems->data + (elems->item_size * index));
         // Due to the checks in aws_cryptosdk_enc_ctx_size, no string can have a length > UINT16_MAX
-        val->key   = ensure_string_is_allocated_bounded_length(UINT16_MAX);
+        __CPROVER_assume(val != NULL);
+        val->key = ensure_string_is_allocated_bounded_length(UINT16_MAX);
+        __CPROVER_assume(val->key != NULL);
         val->value = ensure_string_is_allocated_bounded_length(UINT16_MAX);
+        __CPROVER_assume(val->value != NULL);
```

Comment:
Why can't we assume here `aws_string_is_valid()`?

Reply:
Updated!